DevProxy.Abstractions/Utils/MSGraphDbUtils.cs

```diff
@@ -9,17 +9,18 @@
 
 namespace DevProxy.Abstractions.Utils;
 
-public static class MSGraphDbUtils
+public sealed class MSGraphDbUtils(HttpClient httpClient, ILogger<MSGraphDbUtils> logger) : IDisposable
 {
-    private static readonly Dictionary<string, OpenApiDocument> _openApiDocuments = [];
     private static readonly string[] graphVersions = ["v1.0", "beta"];
-
-    private static SqliteConnection? _msGraphDbConnection;
+    private readonly Dictionary<string, OpenApiDocument> _openApiDocuments = [];
+    private readonly HttpClient _httpClient = httpClient;
+    private readonly ILogger<MSGraphDbUtils> _logger = logger;
+    private SqliteConnection? _msGraphDbConnection;
 
     // v1 refers to v1 of the db schema, not the graph version
     public static string MSGraphDbFilePath => Path.Combine(ProxyUtils.AppFolder!, "msgraph-openapi-v1.db");
 
-    public static SqliteConnection MSGraphDbConnection
+    public SqliteConnection MSGraphDbConnection

```

Comment:
Let's rename to `Connection` since it's an instance member now to simplify it

Reply:
fixed